lib/contact_searcher.rb

```diff
@@ -107,6 +105,12 @@ def api_where(selector=nil)
             end
           when 'first_name', 'last_name'
             self.new_selector[k] = v.is_a?(String)? Regexp.new(v,Regexp::IGNORECASE) : v
+          when 'tags'
```

Comment:
Indent `when` as deep as `case`.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.